spec/controllers/merchants_invoices_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::Merchants::InvoicesController do
+  describe "GET all related invoices" do
+    it "retrieves all invoices related to that merchant" do
+      merchant = FactoryGirl.create(:merchant, id: 1)
+      10.times do
+        FactoryGirl.create(:invoice, merchant: merchant)
+      end
+      
+      get :index, params: { format: :json, id: 1 }
```

Comment:
@concach 
Now that we added that line in ApplicationController to make JSON the default format, do we still need to pass the format in the params here?

Also on line 22 of this file.